src/sql/src/func.rs

```diff
@@ -989,7 +961,12 @@ where
         if candidates == 0 {
             match spec {
                 FuncSpec::Func(name) => PlanError::UnknownFunction {
-                    name: name.to_string(),
+                    name: ecx
+                        .qcx
+                        .scx
+                        .humanize_resolved_name(name)
+                        .expect("resolved to object")
```

Comment:
There's a lot of extra expects on humanize_resolved_name now, and I'm a bit scared of those. Why are they safe?

Reply:
The name resolution pass now guarantees that any function name we receive exists in the catalog.